project-incubation-entry-considerations.md

```diff
@@ -0,0 +1,41 @@
+# Project Incubation Entry Considerations
+This document provides information to help people who are considering bringing their projects to Hyperledger for incubation. Specifically, it will outline items that the TSC will take into consideration when evaluating the project, as well as, some best practices that have been followed by other projects prior to the project proposal being submitted to the TSC. Our hope is that this document will smooth the entry process for new projects being proposed. The [project proposal process](./project-lifecycle.md#proposal) and the [template for project proposals](https://hyperledger.github.io/hyperledger-hip/) are outlined outside of this document. The goal of incubation within Hyperledger is to provide a space for projects that have high potential to grow in the community. Ideas should start in [Hyperledger Labs](https://labs.hyperledger.org/).
```

Comment:
```suggestion
This document provides information to help people who are considering bringing their projects to Hyperledger for [Incubation](./project-lifecycle.md#incubation). Specifically, it will outline items that the TSC will take into consideration when evaluating the project, as well as, some best practices that have been followed by other projects prior to the project proposal being submitted to the TSC. Our hope is that this document will smooth the entry process for new projects being proposed. The [project proposal process](./project-lifecycle.md#proposal) and the [template for project proposals](https://hyperledger.github.io/hyperledger-hip/) are outlined outside of this document. The goal of incubation within Hyperledger is to provide a space for projects that have high potential to grow in the community. Ideas should start in [Hyperledger Labs](https://labs.hyperledger.org/).
```